ufls_detection/ufls_detection_voltage.R

```diff
@@ -0,0 +1,40 @@
+
+ufls_detection_voltage <- function(combined_data, event_time, window_length, 
```

Comment:
I would find it useful to know what 'sort' of variable event_time needs to be. E.g. a string, posix thing, in AEST tz...
I'm not sure whether/how we want to do this?